nano/core_test/node.cpp

```diff
@@ -1085,35 +1086,50 @@ TEST (node, fork_open_flip)
 				 .work (*system.work.generate (key1.pub))
 				 .build_shared ();
 	ASSERT_FALSE (*open1 == *open2);
-	// node1 gets copy that will remain
+
+	// give block open1 to node1, manually trigger an election for open1 and ensure it is in the ledger
 	node1.process_active (open1);
-	node1.block_processor.flush ();
-	node1.block_confirm (open1);
-	// node2 gets copy that will be evicted
-	node2.process_active (open2);
-	node2.block_processor.flush ();
-	node2.block_confirm (open2);
-	ASSERT_EQ (2, node1.active.size ());
-	ASSERT_EQ (2, node2.active.size ());
+	ASSERT_TIMELY (5s, node1.block (open1->hash ()) != nullptr);
+	node1.scheduler.manual (open1);
+	ASSERT_TIMELY (5s, (election = node1.active.election (open1->qualified_root ())) != nullptr);
+	election->transition_active ();
+
+	// create node2, with blocks send1 and open2 pre-initialised in the ledger,
+	// so that block open1 cannot possibly get in the ledger before open2 via background sync
+	system.initialization_blocks.push_back (send1);
+	system.initialization_blocks.push_back (open2);
+	auto & node2 = *system.add_node ();
+	system.initialization_blocks.clear ();
+
+	// give block open2 to node2, manually trigger an election for open2 and ensure it is in the ledger
```

Comment:
Should be "ensure open2 is in node2" instead of "give block open2 to node2". Small typo

Reply:
OK